sale_commission_pricelist/README.rst

```diff
@@ -0,0 +1,73 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+==============================
+Sales commissions by pricelist
+==============================
+
+This module extends the functionality of ... to support ...
```

Comment:
The description is not correct

Reply:
In the PR text you can read "Tests and Readme WIP"
:wink: